ocdskingfisherprocess/cli/commands/transform_collection.py

```diff
@@ -0,0 +1,21 @@
+import ocdskingfisherprocess.database
+import ocdskingfisherprocess.cli.commands.base
+from ocdskingfisherprocess.transform.compile_releases import CompileReleaseTransform
+
+
+class CheckCLICommand(ocdskingfisherprocess.cli.commands.base.CLICommand):
+    command = 'transform-collection'
```

Comment:
That is a really confusing command name it should have at least something todo with checks.

Reply:
I forgot to rename both command classes. Will do that.

>  don't know that we need to suffix -collection to all our commands

This is basically a hangup from the old system. It's one I'm happy to leave for now because another hangup of the old system means that currently, actually getting checks and transforms to run is a kinda annoying process. Currently you can read the command "transform-collection" as "this will transform *one* individual collection" and that's kinda rubbish, we really want some kind of better runner. (One that deals with the points @kindly  raised in his comment too). So I would like to leave the command name as it is for now to make clear the "individual collection" part and come back to this whole issue of how we actually run transforms and checks very soon.